docs/development/profiling.md

```diff
@@ -0,0 +1,75 @@
+# Profiling Flotta Operator
```

Comment:
@masayag What do you think about only documenting how to do it?

Reply:
added: https://github.com/project-flotta/flotta-operator/pull/133